text-generic.md

```diff
@@ -25,6 +25,11 @@ Headings and subheadings never end with a period, even if they happen to sound
 like a real sentence.
 
 ## Lists
+### Numbered Headings
+Numbered headings are formatted with an em dash, like this:
+> #### 1 — First Thing
```

Comment:
Not a huge fan of this, it looks a bit jarring on the page. How about just using "1. First thing"?

Reply:
👍  I'm good with that.  Will update!